chapters/annotations.tex

```diff
@@ -72,6 +73,20 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 <a href="modelica:/MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
 
+\subsubsection{Style Sheet}
+
+A top-level package may inside the \lstinline!Documentation! have an annotation \lstinline!styleSheet="modelica:/Package/Resources/style.css"!, which provides a cascading style sheet.
+It may also be a vector of strings that corresponding to different style sheets that will be cascaded in the given order.
+The given style sheet(s) is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations of the package.
```

Comment:
Ignore singleton case here:
```suggestion
The style sheets are used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations of the package.
```

Reply:
Also, I'd prefer to move this sentence (giving the purpose of the annotation) just after the first one, before the restriction about top-level packages.

Reply:
Will do.